salt/key.py

```diff
@@ -340,6 +344,17 @@ def _get_key_attrs(self, keydir, keyname,
             keysize = self.opts['keysize']
         return keydir, keyname, keysize, user
 
+    def _get_selected_state(self, selected_state=''):
+        if selected_state.startswith('acc'):
```

Comment:
I'm not 100% familiar with this code, but I'm wondering if we want to validate the users input to ensure they typed out the entire word and spit out an error telling them they did not include a valid state if they mispell it, instead of just checking it with `startswith`. what do you think @DmitryKuzmenko since you have been in this code before ? Any chance you can review this PR and what do you think about my suggestion?

Reply:
That if statement was "copied" from line 568. If you want to change it in my PR we should probably change it in both places.

```python
def list_status(self, match):
        '''
        Return a dict of managed keys under a named status
        '''
        acc, pre, rej, den = self._check_minions_directories()
        ret = {}
if match.startswith('acc'):
...
```

Reply:
ahh thanks for pointing out! might not be a concern then. will wait to hear from @DmitryKuzmenko